fcp/node.py

```diff
@@ -883,7 +898,9 @@ def putdir(self, uri, **kw):
         
         #@-node:<<get inventory>>
         #@nl
-        
+
+        # FIXME: This somehow works, but it is borked and
+        # repeated. Clean it up. I bet I am the one responsible...
```

Comment:
sorry … I should have some more time now. Today I’m trying to get the CSS thing done, though.